.github/CODEOWNERS

```diff
@@ -0,0 +1 @@
+* @ExpediaGroup/apiary-committers
```

Comment:
Add a [closing line break](https://stackoverflow.com/questions/5813311/no-newline-at-end-of-file) to EOF please.